.vsts-pr.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
         _configuration: Release
         _testKind: testcoreclr
   steps:
-  - script: ./eng/cibuild.sh --configuration $(_configuration) --$(_testKind)
+  - script: ./eng/cibuild.sh --configuration $(_configuration) --$(_testKind) --verbosity normal
```

Comment:
Remove these to keep the build output cleaner.  `cibuild.sh` and `CIBuild.cmd` publish binary logs anyways which are better.

Reply:
It's not working yet - so need these for debugging

Reply:
I'll remove these once we're green